src/ports/owner/utils.ts

```diff
@@ -24,7 +24,7 @@ export const getOwnersSQLQuery = (
 
   const where = [
     contractAddress
-      ? SQL`nfts.contract_address = ${contractAddress}`
+      ? SQL`left(contract_address, 256) = ${contractAddress}`
```

Comment:
This shouldn't be like this?
```suggestion
      ? SQL`left(nfts.contract_address, 256) = ${contractAddress}`
```

Reply:
works either way, I'm putting the `nfts.` back just to keep it consistent